packages/app/src/h5web/breadcrumbs/BreadcrumbsBar.module.css

```diff
@@ -13,6 +13,7 @@
   justify-content: center;
   overflow: hidden;
   min-width: 0;
+  margin-top: 0;
```

Comment:
I've completely removed some of the styles applied to global elements like `h1`, `p`, `ul`, etc. (mostly margins), so I have to change some styles here and there.